src/Compilers/CSharp/Portable/Syntax/LambdaUtilities.cs

```diff
@@ -47,6 +47,11 @@ public static bool IsLambda(SyntaxNode node)
         public static SyntaxNode GetLambda(SyntaxNode lambdaBody)
         {
             var lambda = lambdaBody.Parent;
+            if (lambda is ArrowExpressionClauseSyntax)
```

Comment:
Thanks!

---
In reply to: [134011291](https://github.com/dotnet/roslyn/pull/21599#discussion_r134011291) [](ancestors = 134011291)